src/components/Explore/explore.css

```diff
@@ -1,6 +1,4 @@
 .explore {
     height: calc(100vh - 56px);
```

Comment:
you can use Relative units instead of px

Reply:
Thank you for pointing it out will change it globally at the end